tools/Signing.Dependencies/Signing.Dependencies.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I don't see this file referenced anywhere.  Does everything under tools/ get picked up for restore or does this need to be added somewhere?